website/content/en/preview/upgrading/upgrade-guide.md

```diff
@@ -32,6 +32,9 @@ kubectl apply -f https://raw.githubusercontent.com/aws/karpenter{{< githubRelRef
 kubectl apply -f https://raw.githubusercontent.com/aws/karpenter{{< githubRelRef >}}pkg/apis/crds/karpenter.sh_nodeclaims.yaml
 kubectl apply -f https://raw.githubusercontent.com/aws/karpenter{{< githubRelRef >}}pkg/apis/crds/karpenter.k8s.aws_ec2nodeclasses.yaml
 ```
+### Upgrading to v0.34.0+
+
+* Karpenter add now by default on `podSecurityContext` level, specific `fsgroup: 65536`. If you are using sidecar containers, you should review if this configuration is compatible for them. If you need to change this default value you have the option adding specific `podSecurityContext:` block on your values.yaml file
```

Comment:
There is already a section labeled `Upgrading to v0.34.0` below, can we just add this as a bullet in that section?

Reply:
Moved to the proper section with [this commit] (https://github.com/aws/karpenter-provider-aws/pull/5314/commits/b39b5c7b67e2239e6310bf0d17369adbc63e06e2)